docker/humble_gzharmonic/dockerfile

```diff
@@ -19,7 +19,15 @@ RUN apt-get install lsb-release wget gnupg && apt-get update \
 && rm -rf /var/lib/apt/lists/*
 
 # Install ros-gazebo dependencies
-RUN sudo apt install ros-humble-ros-gzharmonic -y
+RUN sudo apt update && sudo apt install ros-humble-ros-gzharmonic -y
+
+# Recompile Aerostack2
+RUN rm -rf ~/aerostack2_ws/build ~/aerostack2_ws/install ~/aerostack2_ws/log
+RUN . /opt/ros/$ROS_DISTRO/setup.sh && colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release
+
+RUN echo "source /opt/ros/$ROS_DISTRO/setup.bash" >> ~/.bashrc
```

Comment:
If already using aerostack2 image, this lines are not necessary, as they are duplicated